test/tests/functional/pbs_alps_inventory_check_hook.py

```diff
@@ -131,7 +136,8 @@ def test_cray_login_nodes(self):
         """
         Test log when no nodes with vntype 'cray_login' are present.
         """
-        now = time.time()
+        mc = ""

```

Comment:
Good point.  I saw that.  The real complaint is that self.mom is used before it is set.  I suspect this is a false positive, but just incase, I want to tell @hirenvadalia about it.  He may want to look into it.  I suspect it is nothing.